mne/viz/evoked.py

```diff
@@ -2906,9 +2905,8 @@ def plot_compare_evokeds(
     # cannot combine a single channel
     if (len(picks) < 2) and combine is not None:
         warn(
-            'Only {} channel in "picks"; cannot combine by method "{}".'.format(
-                len(picks), combine
-            )
+            f"Only {len(picks)} channel in 'picks'; cannot combine by method "
+            f"'{combine}'."
```

Comment:
```suggestion
            f'Only {len(picks)} channel in "picks"; cannot combine by method '
            f'"{combine}".'
```

Reply:
Yes I inverted that one, wanted to see if a test would break because of that. We use double brackets `"` everywhere for strings, so it's strange that this one uses the opposite because the `"` were used as part of the string. Personaly, I would fix the test rather than the string.

Reply:
> We use double brackets " everywhere for strings

Actually I think this case is often an exception to that rule:
```
$ git grep -I "'\""
codemeta.json:        "importlib_resources>=5.10.2; python_version<'3.9'",
mne/_fiff/meas_info.py:                        f'"{np.iinfo(">i4").min!r}" and "{np.iinfo(">i4").max!r}", got '
mne/_fiff/meas_info.py:                        f'"{value[key_2]!r}"'
mne/_fiff/meas_info.py:            f'"{(np.iinfo(">i4").min, 0)!r}" and "{(np.iinfo(">i4").max, 0)!r}", got '
mne/_fiff/meas_info.py:            f'"{meas_date_stamp[0]!r}"'
mne/_fiff/meas_info.py:            '"mne.match_channel_orders()" to sort channels.'
...
$ git grep -I "f\"'" | wc -l  # double then immediate single
11
$ git grep -I "f'\"" | wc -l  # single then immediate double
29
```

Reply:
... and
```
$ git grep -I "f'" | wc -l  # f single
319
```
IIRC `black` actually chose (at one point at least, maybe still does?) to format strings this way when double quotes are inside the string

Reply:
Sure there are many exceptions in the current code base. My rationale was that if you had to write an error string including a variable name nowadays, you would use single quotes around the variable name and double quotes around the string, e.g. `"Only {len(picks)} channel in 'picks'"`, *because* black now exist and enforces a style for strings which you got used to. 
IMO, the hits you get are reminiscence of old code prior to the existence of `black` which might even be caught in a `ruff` `UP` rule in the future (maybe already are?). Thus, I would tend to change them little by little when encountered.

Anyway, at the end of the day, it's simply a style choice without any impact on the code, happy to disregard as well.

Reply:
> if you had to write an error string including a variable name nowadays, you would use single quotes around the variable name and double quotes around the string, e.g. "Only {len(picks)} channel in 'picks'", because black now exist and enforces a style for strings which you got used to.

Actually kind of the opposite, I got used to what black does with double quotes inside strings:
```
$ echo 'print("\"hello\"")' > test.py
$ black test.py 
reformatted test.py

All done! ✨ 🍰 ✨
1 file reformatted.
larsoner:~/python/mne-bids-pipeline$ cat test.py 
print('"hello"')
```
and nowadays I tend to use single-quoted strings when I know there will be (or want there to be) double quotes inside the string for whatever reason (aesthetic or functional). Hence why I figure we might as well preserve the ones we already have in the codebase.

> which might even be caught in a ruff UP rule in the future (maybe already are?)

I would be very surprised -- the double-to-single changes you're talking about actually change the *content* of the resulting `str` vars, which I doubt a UP rule would ever target. So far they've just changed things about for example the syntax used to create those vars (e.g., a `%` vs `.format` vs `f`-string all produce the same variable at the end of the day).